### PR TITLE
WIP: Enable sumo marketing popup

### PR DESCRIPTION
**Pull request by arun1595**
##### Description

Enable sumo marketing popup as asked by @PixelantDesign 

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

### CODE REVIEW

**Comment by thelostone-mc** [comment]
@mark / @owocki we good enabling the sumo on the tip flow ?

File: app/dashboard/templates/yge/base.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
   </head>
   <body id="yge" class="{{ class }}">
     {% include 'shared/tag_manager_2.html' %}
-    {% include 'shared/analytics.html' with suppress_sumo=1 %}
+    {% include 'shared/analytics.html' with suppress_sumo=0 %}
```
Comment (thelostone-mc) [inline app/dashboard/templates/yge/base.html]: You can remove the `supress_sumo` just to display it !

Comment (owocki) [reply app/dashboard/templates/yge/base.html]: this is suppressing sumo on the youvegoteth (tips flow).. it doesnt have anything to do with what i perceive to be the issue (sumo not showing on the landing page)

Comment (thelostone-mc) [reply app/dashboard/templates/yge/base.html]: The sumo does appear on the landing page currently though!

**Comment by thelostone-mc** [comment]
@PixelantDesign @arun1595 any reason why we are enabling this on the tip flow ?

**Comment by arun1595** [comment]
@thelostone-mc My bad, let me close this PR as it is not really required.

**Pull request closed without merging**